packages/vgrammar/src/view/mark.ts

```diff
@@ -264,6 +261,21 @@ export class Mark extends GrammarBase implements IMark {
     return this;
   }
 
+  protected evaluateGroup(data: any[]) {
```

Comment:
Why a separate stage of evaluateGroup was split, can't this be merged into current evaluateJoin stage?